packages/sitecore-jss-dev-tools/src/package-deploy.ts

```diff
@@ -1,10 +1,13 @@
 import chalk from 'chalk';
 import fs from 'fs';
-import { Agent as HttpsAgent } from 'https';
+import https, { Agent as HttpsAgent } from 'https';
 import path from 'path';
-import request from 'request';
+import FormData from 'form-data';
+import axios, { AxiosRequestConfig, AxiosError } from 'axios';
 import { TLSSocket } from 'tls';
 import { digest, hmac } from './digest';
+import { ClientRequest } from 'http';
+import { IncomingMessage } from 'http';
```

Comment:
There a reason these imports aren't combined?
```suggestion
import { ClientRequest, IncomingMessage } from 'http';
```

Reply:
Changed